File: Stage3/C/Problem15552.c
#include <stdio.h>

int main() {
    int T, A, B;
    scanf("%d", &T);
    int results[T];

    for (int i = 0; i < T; i++) {
        scanf("%d %d", &A, &B);
        results[i] = A + B;
    }

    for (int i = 0; i < T; i++) {
        printf("%d\n", results[i]);
    }
    
    return 0;
}

File: Stage3/Python/Problem15552.py
import sys

def main():
    input = sys.stdin.readline
    T = int(input())
    results = []
    
    for _ in range(T):
        A, B = map(int, input().rstrip().split())
        results.append(f"{A + B}\n")
    
    sys.stdout.write(''.join(results))
    
if __name__ == "__main__":
    main()